### ISSUE TITLE
Could you share the 10-fold cross-validation (8:1:1) split information for dimensional emotion recognition?

### ISSUE THREAD

**Issue by youcaiSUN** [Xinda-Wu/Werewolf-XL-Baseline#2]

**Comment by Xinda-Wu** [comment]
I used a linear partition way, such as [[(1), (2), (3,..., 10)], [(2), (3), (1, 4,..., 10)],...

**Comment by youcaiSUN** [comment]
Thanks! However, I mean that the exact sample names of three sets for each fold (since some samples in the training and validation set are removed to ensure speaker-independent setting), so that I can make a fair comparison with the results in  the paper. 

Besides, the 'speaker' videos in 'Speaker_Video_20210809.zip' (14635) is not consistent to those (14632) in the label file 'Werewolf-XL_Emotion_labels.xlsx' and paper. Moreover, a few 'speaker' videos (175) could not be found, such as '01_02_385', '01_02_499', and '01_08_333'.

Thanks  very much!

**Comment by youcaiSUN** [comment]
Hi! 
Sorry for bothering you, may I ask that is there any progress?
